index.html

```diff
@@ -1,33 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0"> <!-- displays site properly based on user's device -->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- displays site properly based on user's device -->
 
   <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon-32x32.png">
-  
+  <link rel="stylesheet" href="./css/main.css">
+  <link href="https://fonts.googleapis.com/css2?family=Outfit:wght@300;400;600&display=swap" rel="stylesheet">
   <title>Frontend Mentor | NFT preview card component</title>
-
-  <!-- Feel free to remove these styles or customise in your own stylesheet 👍 -->
-  <style>
-    .attribution { font-size: 11px; text-align: center; }
-    .attribution a { color: hsl(228, 45%, 44%); }
-  </style>
 </head>
-<body>
 
-  Equilibrium #3429
-
-  Our Equilibrium collection promotes balance and calm.
+<body>
+  <section>
+    <div class="line">
+      <div class="card">
+        <div class="main-image-container">
+          <div class="equ-container">
+            <img src="./images/image-equilibrium.jpg" alt="equilibrium" class="equ">
+          </div>
+          <div class="overlay-image-container">
+            <img src="./images/icon-view.svg" alt="view" class="view">
+          </div>
+        </div>
+        <div class="main-information">
+          <div class="main-h">
+            <h1>Equilibrium #3429</h1>
+          </div>
 
-  0.041 ETH
-  3 days left
-  
-  Creation of Jules Wyvern
+          <p class="primary-p">Our Equilibrium collection promotes balance and calm.</p>
+          <div class="secondary-p">
+            <div class="p1">
+              <img src="./images/icon-ethereum.svg" alt="icon" class="eth">
```

Comment:
Whenever dealing with images it is a good idea to generally place them in their own containers so that it is easier to maintain their ratios.

```suggestion
<div class="icon-image-container">
              <img src="./images/icon-ethereum.svg" alt="icon" class="eth">
</div>
```

With this you can now set the container's width and height and set the image's max-width and max-height to 100%. This will force the image to maintain its ration so it doesn't look "Squashed". The css rules will look like this then:

```
.icon-image-container {
  width: 30px; /* Set to own custom width */
  height: 30px; /* Set to own custom height */
}

.icon-image-container > img {
  max-width: 100%;
  max-height: 100%;
}
```